File: answers/jahnvit25/Looping/Q2.cpp
#include<stdio.h>
int main()
{
    int n,i,j;
 
    printf("Enter the number of rows:\n");
    scanf("%d",&n);
 
    for(i=n;i>=1;i--)
    {
        for(j=i;j<=n;j++)
        {
            printf("%c",(char)(j+64));
        }
         printf("\n");
    }
return(0);
}

File: answers/VrishtiSharma/looping/Q2.py
n = int(input("Enter the number of rows : "))
i = 1
while(i<=n):
    x = 65 + n - i
    j = 1
    while(j<=i):
        y = x + j - 1
        print(chr(y), end = "")
        j+=1
    print("\t")
    i+=1
